faraday_middleware.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |spec|
   spec.licenses = ['MIT']
 
   spec.add_dependency 'faraday', ['>= 0.7.4', '< 1.0']
+  spec.add_dependency 'safe_yaml'
```

Comment:
Don't add the dependency here, add it on the Gemfile so the gem won't be installed with Faraday